src/app/api/test-results/api.module.ts

```diff
@@ -2,38 +2,31 @@ import { NgModule, ModuleWithProviders, SkipSelf, Optional } from '@angular/core
 import { Configuration } from './configuration';
```

Comment:
Shouldn't format this document since it's generated

Reply:
reverted